spec/features/users/profile/profile_non_friend_spec.rb

```diff
@@ -19,14 +19,17 @@
     }
   end
 
-  let(:user10) { User.new(user1_params) }
+  let(:user10) { User.new(user10_params) }
 
   context 'when I log in as an authenticated user', :vcr do
+    # See spec/shared_contexts/features/current_user_shared_context.rb for context
+    include_context 'logged in as authenticated user'
+
     before do
-      allow(BackEndFacade).to receive(:get_user).with(@user.id).and_return(@user)
-      allow(BackEndFacade).to receive(:get_profile_user).with(user10.id.to_s).and_return(user10)
-      allow(BackEndFacade).to receive(:get_user_friends).with(@user.id).and_return([])
-      allow(BackEndFacade).to receive(:get_user_friends).with(user10.id).and_return([])
+      allow(UserFacade).to receive(:get_user).with(user.id).and_return(user)
+      allow(UserFacade).to receive(:get_profile_user).with(user10.id.to_s).and_return(user10)
```

Comment:
I checked on this and I think it is necessary to have `.to_s` in the cases where the facade is receiving the params from the request and the `id` is provided in the request as a string.  

If we put an integer as the parameter in the stubbed method, it gets confused and says it was expecting, for example, `"10"` instead of `10`, and it doesn't know how to stub that out for us.  

Alternatively, we might be able to set a default return for the stubbed method when given any parameter, but the way we have it is more specific to our individual test setup and is probably ok keeping it that way.